test/browser/lifecycle.test.js

```diff
@@ -2001,8 +2001,8 @@ describe('Lifecycle methods', () => {
 		});
 
 		it('should be called when child fails in render', () => {
-			// eslint-disable-next-line react/require-render-return
 			class ThrowErr extends Component {
+				// eslint-disable-next-line react/require-render-return
```

Comment:
These changes seem to break the linter:

```txt
/home/travis/build/developit/preact/test/browser/lifecycle.test.js
  2004:4  error  Your render method should have return statement  react/require-render-return
  2500:4  error  Your render method should have return statement  react/require-render-return
```

Reply:
Huh, but that was failing locally though. Hmmm 🤔 back to the drawing board then. Is there a chance we could view the source files from travis?